addons/account/models/account_move.py

```diff
@@ -1234,7 +1234,20 @@ def _compute_tax_totals(self):
                             handle_price_include=False,
                             extra_context={'_extra_grouping_key_': 'epd'},
                         ))
-                move.tax_totals = self.env['account.tax']._prepare_tax_totals(**kwargs)
+
+                amount_untaxed = abs(move.amount_untaxed_signed)
+                amount_total = abs(move.amount_total_signed)
```

Comment:
`abs` is almost never a good idea when when doing accounting, unless you manage the sign another way; but it doesn't seem to be the case here